jest.config.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports = {
+    clearMocks: true,
+    resetMocks: true,
+    coverageProvider: 'v8',
+    coverageReporters: ['html', 'lcov', 'text'],
```

Comment:
This seems to create two folders in coverage, unsure if we can change something.